ansible/roles/xml-app-config/files/deployment-scripts/backend_deployment.yml

```diff
@@ -54,13 +53,13 @@
           remote_src: yes
         loop: "{{ directories }}"
 
-      - name: Copy support scripts to support user home
+      - name: Copy support scripts to app user home
         vars:
           ansible_python_interpreter: /usr/bin/python2.6
         copy:
           src: "/tmp/{{ application_name }}-{{ version }}/{{ supportscripts_file }}"
-          dest: "{{ supportscripts_location }}/"
-          owner: "{{ supportscripts_location | basename }}"
+          dest: "{{ home_dir }}/"
+          owner: "{{ home_dir | basename }}"
           group: chlservices
           mode: preserve
           remote_src: yes
```

Comment:
If you move `supportscripts` to `directories` list then this task is no longer necessary as the previous task will do the move for us.